int8 position_command
---
bool success
int8 position_final